### DISCUSSION TITLE
Version of playwright

### DISCUSSION THREAD

**Discussion by zoe-jobson-IL** [artilleryio/artillery#3362] [Artillery]
I was wondering when artillery might be upgrading the version of playwright to 1.48.0. Our e2e testing and load testing share the same repo, so artillery holds our upgrading playwright back - sand some of the e2e SDETs would like to play with playwrights new websocket stuff.

**Comment by hassy** [comment]
Coming soon via https://github.com/artilleryio/artillery/pull/3363 :)

**Comment by zoe-jobson-IL** [comment]
Thank you much appreciated
